activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -176,13 +218,13 @@ def change_table(table_name, options = {})
           end
         end
 
-        def add_reference(*, **options)
+        def add_reference(table_name, ref_name, **options)
           options[:index] ||= false
           super
         end
         alias :add_belongs_to :add_reference
 
-        def add_timestamps(_, **options)
+        def add_timestamps(table_name, **options)
```

Comment:
Would this not cause warnings of unused arguments?

Reply:
Currently Ruby doesn't raise any warnings of unused method arguments for now.

https://travis-ci.org/rails/rails/jobs/481187001